core/src/main/scala/cromwell/core/DoesNotExpectMessages.scala

```diff
@@ -0,0 +1,10 @@
+package cromwell.core
+
+import akka.actor.{Actor, ActorLogging}
+
+
+trait DoesNotExpectMessages {  self: Actor with ActorLogging =>
```

Comment:
As luck would have it, I was just reading [a blog post on akka logging anti-patterns](http://manuel.bernhardt.io/2016/08/31/akka-anti-patterns-logging-the-wrong-way/) this morning. They specifically mentioned this and pointed people to the built in schemes to handle this stuff: http://doc.akka.io/docs/akka/current/java/logging.html#Auxiliary_logging_options